.github/ISSUE_TEMPLATE/testplan.md

```diff
@@ -30,7 +30,7 @@ as well as an upgrade of the previous version of Teleport.
 
 - [ ] RBAC
 
-  Make sure that invalid and valid attempts are reflected in audit log.
+  Make sure that invalid and valid attempts are reflected in audit log. Do this with both Teleport and Agentless nodes.
```

Comment:
Is there something we can link to to help describe _how_ to do this with agentless nodes?

Reply:
Yes, added a link to agentless docs.